tests/models/mobilebert/test_modeling_tf_mobilebert.py

```diff
@@ -311,15 +311,6 @@ def test_for_token_classification(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_mobilebert_for_token_classification(*config_and_inputs)
 
-    @slow
-    def test_keras_fit(self):
```

Comment:
Just double checking that this is now fast for this model?

Reply:
Actually, good question. On my local machine, this test takes from 20-40 seconds depending on model. MobileBERT is one of the slower ones, but it's still inside that range.

However, 20-40 seconds is probably in the range that the whole test should be marked as slow to keep it out of the quick CI, right?

Reply:
We have some tests (not decorated as slow) run more than 40 seconds, but let's not add more such tests. Decorate it as slow and everyone's life is easier 🍺

Reply:
🍻  cheers to that!